presto-main/src/main/java/com/facebook/presto/tracing/testing/TestingTracerProvider.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.tracing.testing;
+
+import com.facebook.presto.spi.tracing.Tracer;
+import com.facebook.presto.spi.tracing.TracerProvider;
+import com.google.inject.Inject;
+
+public class TestingTracerProvider
+        implements TracerProvider
+{
+    @Inject
```

Comment:
same comments as above; the constructor is directly called. We don't need to inject it.

Reply:
If we want to make it run in integration tests then do we need to inject it?